app/views/content/how-to-apply-for-teacher-training/teacher-training-application.md

```diff
@@ -34,6 +34,13 @@ promo_content:
 navigation: 30.05
 navigation_title: Teacher training application
 navigation_description: Find out what you need to include in your teacher training application and when to apply.
+expander:
+  check-your-qualifications:
+    title: Check your qualifications
+    text: If you're a non-UK citizen and need a visa to come to the UK to train to teach, you need to make sure the course you’re applying for sponsors visas.
+    link_title: Find out more about how to apply for a visa to train to teach in England
```

Comment:
Our link style has a full-stop at the end of the text (for full sentences) but **outside the link** - is this going to be possible to replicate @martyn-w ?

Reply:
Hi @gemmadallmandfe yes that is indeed possible 👍